sormas-backend/src/main/java/de/symeda/sormas/backend/contact/ContactService.java

```diff
@@ -854,20 +854,20 @@ public Predicate createUserFilterWithoutCase(CriteriaBuilder cb, CriteriaQuery c
 		}
 
 		// whoever created it or is assigned to it is allowed to access it
-		Predicate filter = cb.equal(contactPath.join(Contact.REPORTING_USER, JoinType.LEFT), currentUser);
-		filter = cb.or(filter, cb.equal(contactPath.join(Contact.CONTACT_OFFICER, JoinType.LEFT), currentUser));
+		Predicate filter = cb.equal(contactPath.get(Contact.REPORTING_USER).get(User.ID), currentUser.getId());
+		filter = cb.or(filter, cb.equal(contactPath.get(Contact.CONTACT_OFFICER).get(User.ID), currentUser.getId()));
```

Comment:
Isn't Contact Officer optional? Is there a reason that inner instead of left joins are OK?
This may also apply to region and district. Reporting User seems to be required.

Reply:
`contactPath.get(Contact.REPORTING_USER).get(User.ID) ` doesn't generate join, it will user the reportingUser_id field.
Same for contactOfficer. 

This way two join with the user table is avoided.

Reply:
Based on your discussion with Holger, please revert these changes